shell.nim

```diff
@@ -409,3 +509,15 @@ macro shellAssign*(cmd: untyped): untyped =
 
   when defined(debugShell):
     echo result.repr
+
+when isMainModule:
+  var myConfig: set[DebugOutputKind] = {dokError}
```

Comment:
What's the point of `myConfig = {dokError}` if it's not used in the next line and modified to `myConig = {dokOutput}` a few lines below? I guess the next line is just missing the `myConfig` arg?

Also this should also be moved to the tests (or just a test file) imo.